crimethinc/website
# Replace homegrown search with redirect to DuckDuckGo

# What does this pull request do?

This PR replaces our homegrown search with redirect to DuckDuckGo.
Our search had stopped usefully working at some point. Until someone has the time/expertise to rebuild a Postgres centric search, this PR will get our readers/viewers better search results of our site's full archives. 


# How should this be manually tested?

Do a search on /search. Expect to get redirected to DuckDuckGo with your search, with the results scoped to crimethinc.com.
